src/api/app/controllers/webui/project_controller.rb

```diff
@@ -568,13 +568,8 @@ def set_maintained_project
   def load_project_info
     find_maintenance_infos
 
-    # TODO: bento_only, webui2 fetching this in the PackageDatatable model
-    @packages = []
-    @project.packages.order_by_name.pluck(:name, :updated_at).each do |p|
-      @packages << [p[0], p[1].to_i.to_s] # convert Time to epoch ts and then to string
-    end
-
-    @ipackages = @project.expand_all_packages.find_all { |ip| !@packages.map { |p| p[0] }.include?(ip[0]) }
+    @packages = @project.packages.pluck(:name)
+    @ipackages = @project.expand_all_packages.find_all { |ip| !@packages.include?(ip[0]) }
```

Comment:
What is `i` in `@ipackages`? Is it `included`?

Reply:
inherited_pacakges

Reply:
Could we rename the variable please?

Reply:
sure

Reply:
done!